pymaker/oasis.py

```diff
@@ -610,7 +610,7 @@ def add_token_pair_whitelist(self, base_token: Address, quote_token: Address) ->
         return Transact(self, self.web3, self.abi, self.address, self._contract,
                         'addTokenPairWhitelist', [base_token.address, quote_token.address])
 
-    def get_orders(self, pay_token: Address = None, buy_token: Address = None) -> List[Order]:
+    def get_orders(self, pay_token: Address = None, p_token: Token = None,  buy_token: Address = None, b_token: Token = None) -> List[Order]:
```

Comment:
since Token is already instantiated with an Address I suggest signature of this method to be
`def get_orders(self, pay_token: Token = None, buy_token: Token = None) -> List[Order]:`

and to use pay_token.address / use buy_token.address for retrieving Token address